samples/sample_encode/include/pipeline_encode.h

```diff
@@ -180,6 +180,9 @@ struct sInputParams
 
 #if (MFX_VERSION >= 1027)
     msdk_char *RoundingOffsetFile;
+#endif
+#if (MFX_VERSION >= 1028)
```

Comment:
MFX_VERSION_NEXT here and below

Reply:
Hi Dmitry, when you upgrade MFX_VERSION to 1029, will you replace all current MFX_VERSION_NEXT with 1028. I think this feature should be started from version 1028, so I use 1028 here.